server/src/test/java/org/opensearch/index/translog/RemoteFsTranslogTests.java

```diff
@@ -671,13 +679,13 @@ public void testSimpleOperationsUpload() throws Exception {
             assertThat(snapshot.totalOperations(), equalTo(ops.size()));
         }
 
-        assertEquals(2, translog.allUploaded().size());
+        assertEquals(4, translog.allUploaded().size());
```

Comment:
These changes are reverting back to previous values. We need to change this for combined class with pinned timestamp. As we are abstracting out implementation for pinned timestamp, reverting these back to original values.